### PR TITLE
OSOE-53: RouteModel, unit test fixes, orchard-core-preview

### PR DESCRIPTION
**Pull request by sarahelsaig**
[OSOE-53](https://lombiq.atlassian.net/browse/OSOE-53)
Depends on OSOE-52.
Root PR: https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/pull/25

### CODE REVIEW

File: Lombiq.Tests.UI/Extensions/ShortcutsUITestContextExtensions.cs [hunk 1]
```diff
@@ -35,7 +30,7 @@ public static class ShortcutsUITestContextExtensions
         /// enabled.
         /// </summary>
         public static void SignInDirectly(this UITestContext context, string userName = DefaultUser.UserName) =>
-                GoTo<AccountController>(context, nameof(AccountController.SignInDirectly), (nameof(userName), userName));
+            context.GoTo<AccountController>(controller => controller.SignInDirectly(userName));
```
Comment (Piedone) [inline Lombiq.Tests.UI/Extensions/ShortcutsUITestContextExtensions.cs:33]: We get these:

image

File: Lombiq.Tests.UI/Services/UITestContext.cs [hunk 2]
```diff
@@ -53,6 +53,11 @@ public class UITestContext
         /// </summary>
         public Dictionary<string, object> CustomContext { get; } = new();
 
+        /// <summary>
+        /// Gets or sets the current tenant name when testing multi-tenancy.
+        /// </summary>
+        public string TenantName { get; set; }
```
Comment (Piedone) [inline Lombiq.Tests.UI/Services/UITestContext.cs]: This is never set.

**Review by Piedone — CHANGES_REQUESTED** [review]

File: Lombiq.Tests.UI/Services/UITestContext.cs (hunk 2 above)
Comment (sarahelsaig) [reply Lombiq.Tests.UI/Services/UITestContext.cs]: Yes, it's just a placeholder. I'm not experienced with multitenancy but I assume we will want to be able to test such sites at some point in the future? At that point this prop could be set by the test.

Comment (Piedone) [reply Lombiq.Tests.UI/Services/UITestContext.cs]: Set it to `"Default"` here then please, and make a note about why is it like that there and here: https://github.com/Lombiq/UI-Testing-Toolbox/issues/80

File: Lombiq.Tests.UI/Extensions/ShortcutsUITestContextExtensions.cs (hunk 1 above)
Comment (sarahelsaig) [reply Lombiq.Tests.UI/Extensions/ShortcutsUITestContextExtensions.cs:33]: Any idea why it doesn't show up in CI?

Comment (Piedone) [reply Lombiq.Tests.UI/Extensions/ShortcutsUITestContextExtensions.cs:33]: .NET SDK warnings have this issue for some reason, but not always...

**Pull request merged**